### ISSUE TITLE
web-ext run no longer works with adb, running Arch Linux

### ISSUE THREAD

**Issue by kalinda99** [mozilla/web-ext#1709]
I installed this on my system from here: https://aur.archlinux.org/packages/nodejs-web-ext/ It was built using yarn but it's been working fine.

### Is this a feature request or a bug?
A bug? It weirdly just started and I've been running version 3.1.1 for a while now without issue.

### What is the current behavior?
When I run `web-ext run --target=firefox-android --android-device=[my google pixel 2] --firefox-apk=org.mozilla.firefox` it fails to work and spits out this error at me:

````
FailError: Failure: ''
 at /usr/lib/node_modules/web-ext/node_modules/adbkit/lib/adb/parser.js:183:29
at tryCatcher (/usr/lib/node_modules/web-ext/node_modules/bluebird/js/main/util.js:26:23) 
at Promise._settlePromiseFromHandler (/usr/lib/node_modules/web-ext/node_modules/bluebird/js/main/promise.js:503:31)
at Promise._settlePromiseAt (/usr/lib/node_modules/web-ext/node_modules/bluebird/js/main/promise.js:577:18)
at Async._drainQueue (/usr/lib/node_modules/web-ext/node_modules/bluebird/js/main/async.js:128:12)
at Async._drainQueues (/usr/lib/node_modules/web-ext/node_modules/bluebird/js/main/async.js:133:10)
at Immediate.Async.drainQueues [as _onImmediate] (/usr/lib/node_modules/web-ext/node_modules/bluebird/js/main/async.js:15:14)
at runCallback (timers.js:705:18)
at tryOnImmediate (timers.js:676:5)
at processImmediate (timers.js:658:5)
````

This happens with my WIP addon and another one I've got that I haven't touched in ages and that worked fine with this last time I tried it. This behaviour happens on two computers, both running Arch.

### What is the expected or desired behavior?
It launches without issue and I test my addon on my phone.

### Version information (for bug reports)

* **Firefox version**: 68.1.1 on Android 9 (which I recently updated to)
* **Your OS and version**: Arch Linux, kernel 5.2.11
* Paste the output of these commands: 
````
node --version && npm --version && web-ext --version
v10.16.3
6.11.3
3.1.1
````

Any help appreciated, thanks!

**Comment by rpl** [comment]
@kalinda99 we have a couple of questions that may help us to reproduce the issue and investigate it: 
- do you know if the issue is also happening on android versions < 9?
- do you have adb binary installed? (if you do, and which version are you using?)
- can you reproduce the issue while running web-ext with the --verbose option and provide us the more verbose logs produced?

Based on the stacktrace in the issue description, the issue is happening during adbkit parsing, and so it would be also useful to collect more verbose logs from adbkit dependency too. You should be able to force adbkit to produce this additional logs using:

```
$ DEBUG=adb:* web-ext --verbose ...
```
Thanks!

**Comment by kalinda99** [comment]
Thanks for the reply! I can confirm that it also doesn't work with Android 8 either, and gives the same error, since I tried it on a different device. It also failed with my friend's Galaxy Note 9 running Android 9, too.

I'm using adb from Arch Linux's [android-tools](https://www.archlinux.org/packages/community/x86_64/android-tools/) package, which is the latest in the repos at version 29.0.3. I could try adb off Google's website though.

I've attached a copy of the output from the debug commands: 
[web-ext_debug.txt](https://github.com/mozilla/web-ext/files/3660306/web-ext_debug.txt)

Many thanks~
